runtime/compiler/env/j9methodServer.hpp

```diff
@@ -88,7 +88,7 @@ class TR_J9MethodFieldAttributes
       if (result) *result = _result;
       }
 
-   void *_fieldOffsetOrAddress; // stores uint32_t for non-static attributes
+   uintptr_t _fieldOffsetOrAddress; // stores uint32_t for non-static attributes, pointer otherwise
```

Comment:
Small nitpick, but can we improve this comment? Perhaps something like:
```
Stores a uint32_t representing an offset for non-static fields, or an address for static fields.
```

Reply:
Ok